components/pedigree/resources/src/main/resources/pedigree/model/export.js

```diff
@@ -259,7 +259,7 @@ define([
               var rejectedCandidateGenes  = [];
               var carrierGenes   = [];
               nodeGenes.each( function(item) {
-                  if (candidateGeneSelected && (item.status == "candidate")) {
+                  if (candidateGeneSelected && (item.status == "candidate" || item.status.startsWith("candidate>")) {
```

Comment:
This whole block (starting at line 256) needs to be reworked so that the statuses are not hardcoded.